pippo-session-ehcache2/src/main/java/ro/pippo/session/ehcache/EhcacheInitializer.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016 Herman Barrantes.
```

Comment:
I prefer the standard line `Copyright (C) 2014 the original author or authors.`. 
You are the author of these files and I think that my propose is the same thing.
Do you see some problem with my propose?